pkg/autoscaler/fake/fake_metric_client.go

```diff
@@ -17,9 +17,28 @@ limitations under the License.
 package fake
 
 import (
+	"fmt"
 	"time"
 
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
+	kubeinformers "k8s.io/client-go/informers"
+	fakeK8s "k8s.io/client-go/kubernetes/fake"
+)
+
+var (
+	// KubeClient holds instances of interfaces for making requests to kubernetes client.
+	KubeClient = fakeK8s.NewSimpleClientset()
+	// KubeInformer constructs a new instance of sharedInformerFactory for all namespaces.
+	KubeInformer = kubeinformers.NewSharedInformerFactory(KubeClient, 0)
+)
+
+const (
+	// TestRevision is the name used for revision
```

Comment:
nit: comments should be sentences and end in periods.

```suggestion
	// TestRevision is the name used for the revision.
```